utils/bcd.js

```diff
@@ -0,0 +1,17 @@
+let requireId = '..';
+
+if (process.env.BCD_UTILS_BCD_PACKAGE_PATH) {
+  console.warn('Warning: loading BCD from alternate path');
+  requireId = process.env.BCD_UTILS_BCD_PACKAGE_PATH;
+}
+
+if (process.env.BCD_UTILS_REQUIRE_PACKAGE) {
+  // To facilitate further experimentation while also not exposing this as part
+  // of BCD's public API. With this, you can use `/utils` as a submodule.
+  console.warn('Warning: `require`-ing BCD');
+  requireId = '@mdn/browser-compat-data';
+}
```

Comment:
Unless and until we expose this a public API, you'd have to copy the `utils` to use them independently of the BCD package. This ought to enable a slightly more ergonomic hack around that problem.

Reply:
`query` allows passing in the BCD copy to use, which solves this and is convenient for testing. If we do that consistently, do we still need this mechanism?

Reply:
That's great idea. I'll flag the couple of places that need this.

Reply:
OK, I've added a `data` parameter to all of the utils where that made sense. This made it possible to drop this module entirely.